nvm.sh

```diff
@@ -1089,6 +1089,7 @@ nvm_ls() {
     esac
 
     nvm_is_zsh && setopt local_options shwordsplit
+    nvm_is_zsh && unsetopt local_options markdirs
```

Comment:
This will need a regression test that sets these opts when the shell is zsh

Reply:
This has been added, should be called automatically now with other fast tests, but will see in a minute once travis finishes.